### PR TITLE
Upgrade to Butterknife 8.0.1

### PR DESCRIPTION
**Pull request by Plastix**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -134,6 +134,7 @@ dependencies {
     compile libraries.supportCardView
 
     compile libraries.butterKnife
+    apt libraries.butterKnifeCompiler
```
Comment (vanniktech) [inline app/build.gradle]: I'd add a new line after this one

**Comment by vanniktech** [comment]
Is it really necessary to unbind everywhere even in Activitys? AFAIK with Butterknife 7 it was only necessary in Fragments.

**Comment by Plastix** [comment]
@vanniktech I'm not sure. I was doing that just to be safe.

**Comment by jug6ernaut** [comment]
Unbinding is only necessary in fragments. Unbinding is automatic for view listeners in activities.

File: dependencies.gradle [hunk 2]
```diff
@@ -25,7 +25,7 @@ ext.versions = [
         retrofit                     : '2.0.0-beta4',
         jackson                      : '2.6.3',
         autoValue                    : '1.2-rc1',
-        butterKnife                  : '7.0.1',
+        butterKnife                  : '8.0.0',
```
Comment (artem-zinnatullin) [inline dependencies.gradle]: 8.0.1

**Comment by Plastix** [comment]
@artem-zinnatullin Findbugs has been complaining that the Butterknife Unbinder objects are not initialized in a constructor. How should I tell Findbugs to ignore them?

**Comment by vanniktech** [comment]
That's what works for me, dunno if there's a simpler solution:

```xml
<Match>
    <Class name="~.*\$\$ViewBinder*"/>
</Match>
<Match>
    <Class name="~.*\$\$ViewBinder\$InnerUnbinder*"/>
</Match>
```

**Comment by artem-zinnatullin** [comment]
Still fails

**Comment by Plastix** [comment]
I haven't fixed the Find bugs issues. It @vanniktech's solution the way to go?

**Comment by artem-zinnatullin** [comment]
I'm ok with it

**Comment by Plastix** [comment]
@artem-zinnatullin CIs are being flaky. Everything passes on my system: https://gist.github.com/Plastix/5757dda35919f36c8931fb76b272593f

**Comment by artem-zinnatullin** [comment]
Yeah, aapt failed.. Restarted.

**Comment by artem-zinnatullin** [comment]
@vanniktech, @lenguyenthanh PTAL

**Comment by vanniktech** [comment]
👍

**Comment by lenguyenthanh** [comment]
:+1: should we include change log to the description? @Plastix

**Comment by artem-zinnatullin** [comment]
Usually it's better to have links nearby to see what should you expect from the update.

Thanks, @Plastix!

**Pull request merged**